joy_to_servo_pub:
  ros__parameters:
    joy_topic: "/joy"
    twist_topic: "/servo_node/delta_twist_cmds"
    base_frame_id: "panda_link0" # Will need to change to the arm's actual base frame